source/common/http/http1/codec_impl.cc

```diff
@@ -1139,9 +1139,9 @@ int ClientConnectionImpl::onHeadersComplete() {
     }
 
     if (parser_.status_code == 100) {
-      // http-parser treats 100 continue headers as their own complete response.
-      // Swallow the spurious onMessageComplete and continue processing.
-      ignore_message_complete_for_100_continue_ = true;
+      // http-parser treats 1xx headers as their own complete response. Swallow the spurious
+      // onMessageComplete and continue processing.
+      ignore_message_complete_for_1xx_continue_ = true;
```

Comment:
Good point. It's unclear why it worked before (seemed to even pass some CI), but now that I run it after merging to HEAD and adjusting for the new/legacy codec implementations, I see a failure in H1 `allocHeaders()` on the multi-1xx `protocol_integration_test`, when we process the 100 after the 102, since the stream is reset by the 102. Will fix.